smart-contracts/src/watcher/watcher.ts

```diff
@@ -1,94 +1,107 @@
-import { filter, map } from "rxjs/operators"
-import { connectable, interval, merge, Observable, ReplaySubject, Subscription } from "rxjs"
-import { isNotNullOrUndefined, jsonParseSimple, tailFileAsObservable } from "./utilities"
-import { EbRelayerEvent, toEvmRelayerEvent } from "./ebrelayer"
-import { SifnodedEvent, toSifnodedEvent } from "./sifnoded"
-import { HardhatRuntimeEnvironment } from "hardhat/types"
-import { BridgeBank, CosmosBridge } from "../../build"
+import {filter, map} from "rxjs/operators"
```

Comment:
Nit: I think you have the wrong linter setup

Reply:
Done